src/components/Tracker.module.css

```
@@ -0,0 +1,40 @@
+.tracker {
+    height: 170px;
+    background-color: #ffffff;
+    margin-left: 46px;
+    margin-right: 46px;
+    border-radius: 10px;
+    padding: 18px 16px;
+    position: relative;
+}
```

```suggestion
    overflow-x: auto;
}
```
박스 밖으로 넘치는 트래커는 가로 스크롤로 바꾸면 어떨까요?

---

반영하겠습니다!